scripts/utils.js

```diff
@@ -0,0 +1,5 @@
+export const initVanillaFramework = () => {
```

Comment:
Is this needed if we don't need it to work from browser global context?

Reply:
It's not, but I thought we might want to keep if for the cjs version? I'll scrap it from this PR which is module focused

Reply:
Thanks. It's good if it's that easy to add cjs version if we want to. Lets keep this one minimal. It easier to add later than to remove.